### ISSUE TITLE
Loss functions comprehension used in work

### ISSUE THREAD

**Issue by VladyslavML2022** [fenglinglwb/MAT#45]
Hi!
Thank you for the great work!

Currently, I am going to finetune your Places model on my own dataset and try to understand the loss functions that use in your solution.

Looking at the tensorboard dashboard, I saw a lot of charts that can be confusing for me.
Could you explain loss functions which we can see in tensorboard dashboard (specifically Loss/D/loss, Loss/D/loss_s1, Loss/D/reg, Loss/D/reg_s1, Loss/G/l1_loss, Loss/G/loss, Loss/G/loss_s1, Loss/G/pcp_loss, Loss/r1_penalty, Loss/r1_penaly_s1, Loss/scores/fake, Loss/scores/fake_s1,  Loss/scores/real, Loss/scores/real_s1, Loss/signs/fake, Loss/signs/fake_s1, Loss/signs/real, Loss/signs/real_s1) in simple words.

As I figure out, after some training progress, the process should be convergence. However, so far, I cannot understand what are we minimizing and what are we maximizing. 

1) What simple interpretation of these loss functions?
2) What are the key points to pay attention to?
3) If say simply, which loss should fall and which should rise?
4) Particularly, how should loss/scores/fake and loss/real/fake behave themselves?

I will be grateful for your answer. Thanks in advance!

**Comment by fenglinglwb** [comment]
Please provide an email so that I can send the tensorboard log file to you. But only the file for FFHQ is now available.

**Comment by VladyslavML2022** [comment]
reshetnyakvlad2018@gmail.com
Thanks a lot!

**Comment by VladyslavML2022** [comment]
> ###

reshetnyakvlad2018@gmail.com
Thanks a lot!
I hope that helps me in comprehension.
